web/public/images/electricitymap-icon.svg

```diff
@@ -1,4 +1,11 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="117" height="117" viewBox="0 -10 117 117">
-  <path d="M10.29 12.042 0 95.742l16.4-5.998 5.279-10.581L9.558 83.66l8.014-65.187 16.488-4.149 16.486 5.652 7.003-6.064-23.139-7.94zm78.03 3.909 12.111 4.019 7.122 63.894-22.261-8.931-25.25 10.627-8.123-3.453-7.584 5.483 15.692 6.668 25.338-10.663L117 96.285l-9.174-82.303-16.525-5.484z"/>
-  <path d="m84.842 1.582-51.148 46.87c-1.225 1.121-1.038 2.677.412 3.479l7.957 4.404a2.875 2.875 0 0 1 1.084 4.025L24.918 90.789c-.85 1.42-.468 1.755.857.747l54.968-41.853c1.336-1.017 1.156-2.341-.376-2.977l-10.482-4.348a2.823 2.823 0 0 1-1.473-3.869L85.753 2.271c.428-.893.457-1.364.154-1.364-.207 0-.569.22-1.065.675Z"/>
-</svg>
\ No newline at end of file
+<svg width="512" height="512" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M419.859 438.95C430.035 438.95 438.285 430.403 438.285 419.859C438.285 409.315 430.035 400.768 419.859 400.768C409.683 400.768 401.434 409.315 401.434 419.859C401.434 430.403 409.683 438.95 419.859 438.95Z" fill="black"/>
+<path d="M256 438.95C266.176 438.95 274.425 430.403 274.425 419.859C274.425 409.315 266.176 400.768 256 400.768C245.824 400.768 237.574 409.315 237.574 419.859C237.574 430.403 245.824 438.95 256 438.95Z" fill="black"/>
+<path d="M92.1407 438.95C102.317 438.95 110.566 430.403 110.566 419.859C110.566 409.315 102.317 400.768 92.1407 400.768C81.9645 400.768 73.7151 409.315 73.7151 419.859C73.7151 430.403 81.9645 438.95 92.1407 438.95Z" fill="black"/>
+<path d="M419.859 275.091C430.035 275.091 438.285 266.544 438.285 256C438.285 245.456 430.035 236.909 419.859 236.909C409.683 236.909 401.434 245.456 401.434 256C401.434 266.544 409.683 275.091 419.859 275.091Z" fill="black"/>
+<path d="M92.1407 275.091C102.317 275.091 110.566 266.544 110.566 256C110.566 245.456 102.317 236.909 92.1407 236.909C81.9645 236.909 73.7151 245.456 73.7151 256C73.7151 266.544 81.9645 275.091 92.1407 275.091Z" fill="black"/>
+<path d="M419.859 111.232C430.035 111.232 438.285 102.685 438.285 92.1408C438.285 81.597 430.035 73.0496 419.859 73.0496C409.683 73.0496 401.434 81.597 401.434 92.1408C401.434 102.685 409.683 111.232 419.859 111.232Z" fill="black"/>
+<path d="M256 111.232C266.176 111.232 274.425 102.685 274.425 92.1408C274.425 81.597 266.176 73.0496 256 73.0496C245.824 73.0496 237.574 81.597 237.574 92.1408C237.574 102.685 245.824 111.232 256 111.232Z" fill="black"/>
+<path d="M92.1407 111.232C102.317 111.232 110.566 102.685 110.566 92.1408C110.566 81.597 102.317 73.0496 92.1407 73.0496C81.9645 73.0496 73.7151 81.597 73.7151 92.1408C73.7151 102.685 81.9645 111.232 92.1407 111.232Z" fill="black"/>
+<path d="M344.736 227.405H267.066V159.846C267.066 152.384 257.933 149.152 253.53 155.059L161.28 272C157.44 277.12 160.979 284.614 167.258 284.614H244.934V352.154C244.934 359.616 254.067 362.848 258.47 356.941L350.72 240C354.56 234.88 351.014 227.405 344.736 227.405Z" fill="black"/>
```

Comment:
Looks like there are 8 perfect circles that could be simplified with [`<circle>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/circle) here.
Not blocking by any means but it would make the file a bit smaller.

Reply:
Good point also the paths could be joined as there's no reason to be able to color them individually. However, I am not certain it works with our icon-sprite https://github.com/electricitymap/electricitymap-contrib/blob/e01f89daa8fef978574094333e28d411c4d80a6b/web/public/images/icon-sprite.svg?short_path=83ed875#L29

Reply:
Everything that is valid SVG should work in the sprite but I can take a look at it tomorrow if you want.

Reply:
Yeah that's a fun idea :)